src/gs-updates-page.c

```diff
@@ -646,6 +650,15 @@ _get_app_sort_key (GsApp *app)
 	 * 4. online device firmware */
 	g_string_append_printf (key, "%u:", _get_app_section (app));
 
+	switch (gs_app_get_state (app)) {
+	case AS_APP_STATE_INSTALLING:
+		g_string_append (key, "1:");
+		break;
+	default:
+		g_string_append (key, "2:");
+		break;
```

Comment:
Sorry to only notice it now, but if it's just one state, can you make this an if statement instead?